lib/otw_sanitize/embed_sanitizer.rb

```diff
@@ -7,18 +7,19 @@ module OtwSanitize
   # Creates a Sanitize transformer to sanitize embedded media
   class EmbedSanitizer
     ALLOWLIST_REGEXES = {
-      archiveorg:       %r{^archive\.org\/embed/},
-      bilibili:         %r{^(player\.)?bilibili\.com/},
-      criticalcommons:  %r{^criticalcommons\.org/},
-      eighttracks:      %r{^8tracks\.com/},
-      google:           %r{^google\.com/},
-      podfic:           %r{^podfic\.com/},
-      soundcloud:       %r{^(w\.)?soundcloud\.com/},
-      spotify:          %r{^(open\.)?spotify\.com/},
-      viddersnet:       %r{^vidders\.net/},
-      viddertube:       %r{^viddertube\.com/},
-      vimeo:            %r{^(player\.)?vimeo\.com/},
-      youtube:          %r{^youtube(-nocookie)?\.com/}
+      "4shared": %r{^(w\.)?4shared\.com/web/embed},
```

Comment:
```suggestion
      "4shared": %r{^4shared\.com/web/embed},
```
The `www` is already stripped in `standardize_url`. This would match `w.4shared.com/...`, which is incorrect.